<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArchitectureScriptEditor</class>
 <widget class="QWidget" name="ArchitectureScriptEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Architecture Script Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="mpTypeSelector"/>
     </item>
     <item>
      <widget class="QPushButton" name="mpAddButton">
       <property name="text">
        <string>add</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mpRemoveButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../auxiliaries/gui/icons/cedar_aux_gui_icons.qrc">
         <normaloff>:/cedar/auxiliaries/gui/trashcan.svg</normaloff>:/cedar/auxiliaries/gui/trashcan.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableWidget" name="mpScriptList">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>controls</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>type</string>
       </property>
      </column>
     </widget>
     <widget class="cedar::aux::gui::Configurable" name="mpConfigurationEditor" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cedar::aux::gui::Configurable</class>
   <extends>QWidget</extends>
   <header>cedar/auxiliaries/gui/Configurable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../auxiliaries/gui/icons/cedar_aux_gui_icons.qrc"/>
  <include location="../../devices/gui/cedar_dev_gui_icons.qrc"/>
  <include location="icons/cedar_proc_gui_icons.qrc"/>
 </resources>
 <connections/>
</ui>
